docs/conf.py

```diff
@@ -39,9 +39,14 @@ def get_version():
     'recommonmark',
     'notfound.extension',
     'sphinx_search.extension',
+    'hoverxref.extension',
 ]
 templates_path = ['_templates']
 
+hoverxref_auto_ref = True
+# hardcode version because external versions does not work properly
```

Comment:
Why don't they work properly?

Reply:
Because the version slug is the number of the PR. In this case, `6080`.

So, the URL https://readthedocs.org/api/v2/embed/?project=docs&version=6080&doc=config-file/v2&section=Migrating%20from%20v1 returns a 404

but using `latest` works.

https://readthedocs.org/api/v2/embed/?project=docs&version=latest&doc=config-file/v2&section=Migrating%20from%20v1

I didn't check the backend yet to know if this can be fixed.